src/styles/themes/light.ts

```diff
@@ -62,7 +63,7 @@ const lightTheme = {
     heroCard: colors.blue400,
     uploadPreviewActivityIndicator: colors.lightHighlightBackground,
     dropUIBG: 'rgba(252, 251, 249, 0.92)',
-    dropTransparentOverlay: 'rgba(255,255,255,0)',
+    receiptDropUIBG: '', // TODO: add color
```

Comment:
Are we going to merge with the TODO?
```suggestion
    receiptDropUIBG: '', // TODO: add color
```

Reply:
There is an ongoing Theme migration to Typescript [PR](https://github.com/Expensify/App/pull/24178/files), they marked `receiptDropUIBG` as optional. They also typed themes a bit differently, we need to coordinate these two PRs @fabioh8010, wdyt?

Reply:
[asked if there is proper coordination between these two PRs](https://expensify.slack.com/archives/C0593T50UHG/p1696828551876049)